### PR TITLE
reducing number of tooling tests to one

### PR DESCRIPTION
**Pull request by sourabhsinghs**
* updated tooling tests to run single test

### CODE REVIEW

File: tools/distrib/install_python_modules_and_run_tests.sh [hunk 1]
```diff
@@ -22,4 +22,4 @@ cd "$BASEDIR";
 ./tools/distrib/install_all_python_modules.sh
```
[the remark this answers is not in the document]
Comment (XuanWang-Amos) [reply tools/distrib/install_python_modules_and_run_tests.sh:22]: +1

File: src/python/grpcio_tests/tests/unit/test_all_modules_installed.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import grpcio_health_checking
+import grpcio_reflection
+import grpcio_status
+import grpcio_testing
+import grpcio_csm_observability
+import grpcio_tools
+
+class TestAllModulesInstalled(unittest.TestCase):
+    def test_import_all_modules(self):
+        # This test simply imports all the modules.
```
Comment (gnossen) [inline src/python/grpcio_tests/tests/unit/test_all_modules_installed.py:29]: I'd suggest moving the import statements to this scope so the test will fail in the usual manner.

File: src/python/grpcio_tests/tests/unit/test_all_modules_installed.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import unittest
+import logging
+import grpcio_channelz
+import grpcio_csds
+import grpcio_admin
+import grpcio_health_checking
+import grpcio_reflection
+import grpcio_status
+import grpcio_testing
+import grpcio_csm_observability
```
Comment (XuanWang-Amos) [inline src/python/grpcio_tests/tests/unit/test_all_modules_installed.py:24]: Can we add `import grpc_observability` to the list?

**Review by gnossen — APPROVED** [review]

File: src/python/grpcio_tests/tests/unit/test_all_modules_installed.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+import unittest
+import logging
+import grpcio_channelz
+import grpcio_csds
+import grpcio_admin
+import grpcio_health_checking
+import grpcio_reflection
+import grpcio_status
+import grpcio_testing
```
Comment (XuanWang-Amos) [inline src/python/grpcio_tests/tests/unit/test_all_modules_installed.py:23]: I'm wondering if we need test `grpcio_testing`, @gnossen any thoughts?

Comment (gnossen) [reply src/python/grpcio_tests/tests/unit/test_all_modules_installed.py:23]: I think we can get rid of it.

File: src/python/grpcio_tests/tests/unit/test_all_modules_installed.py (hunk 2 above)
Comment (sourabhsinghs) [reply src/python/grpcio_tests/tests/unit/test_all_modules_installed.py:28]: moved

File: src/python/grpcio_tests/tests/unit/test_all_modules_installed.py (hunk 3 above)
Comment (sourabhsinghs) [reply src/python/grpcio_tests/tests/unit/test_all_modules_installed.py]: added

**Pull request closed without merging**